main.go

```diff
@@ -37,7 +35,6 @@ import (
 	policyv1 "open-cluster-management.io/cert-policy-controller/api/v1"
 	controllers "open-cluster-management.io/cert-policy-controller/controllers"
 	"open-cluster-management.io/cert-policy-controller/version"
-	//+kubebuilder:scaffold:imports
```

Comment:
I wasn't sure what this kubebuilder annotation did exactly, it's poorly documented, it's not in our other controllers, and removing it didn't seem to affect anything, so I went ahead with removing the gci formatting exception for this file in the Makefile, which took it out completely.